### ISSUE TITLE
Icon Request

### ISSUE THREAD

**Issue by dalaeance** [Donnnno/Arcticons#56]
I really love your icon pack, and appreciate your taking icon requests. I know this is a pretty sizeable request, so I'd like to emphasize that there's no rush, though I'm sure you're going at a pace comfortable for you.

[20210207_031419.zip](https://github.com/Donnnno/Arcticons/files/5939878/20210207_031419.zip)

```
<!-- Barinsta -->
<item component="ComponentInfo{me.austinhuang.instagrabber/awais.instagrabber.activities.MainActivity}" drawable="barinsta"/>
<!-- Calculator++ (Window mode) -->
<item component="ComponentInfo{org.solovyev.android.calculator/org.solovyev.android.calculator.floating.FloatingCalculatorActivity}" drawable="calculatorwindowmode"/>
<!-- Chromium -->
<item component="ComponentInfo{org.chromium.chrome/com.google.android.apps.chrome.Main}" drawable="chromium"/>
<!-- COVID Alert -->
<item component="ComponentInfo{ca.gc.hcsc.canada.stopcovid/app.covidshield.MainActivity}" drawable="covidalert"/>
<!-- Feeel -->
<item component="ComponentInfo{com.enjoyingfoss.feeel/io.flutter.embedding.android.FlutterActivity}" drawable="feeel"/>
<!-- Health Mate -->
<item component="ComponentInfo{com.withings.wiscale2/com.withings.wiscale2.MainActivity}" drawable="healthmate"/>
<!-- Home Assistant -->
<item component="ComponentInfo{io.homeassistant.companion.android.minimal/io.homeassistant.companion.android.launch.LaunchActivity}" drawable="homeassistant"/>
<!-- LocationPrivacy -->
<item component="ComponentInfo{info.guardianproject.locationprivacy/info.guardianproject.locationprivacy.MainActivity}" drawable="locationprivacy"/>
<!-- Maps Go -->
<item component="ComponentInfo{com.google.android.apps.mapslite/org.chromium.webapk.shell_apk.MainActivity}" drawable="mapsgo"/>
<!-- My Expenses -->
<item component="ComponentInfo{org.totschnig.myexpenses/org.totschnig.myexpenses.activity.SplashActivity}" drawable="myexpenses"/>
<!-- Noice -->
<item component="ComponentInfo{com.github.ashutoshgngwr.noice/com.github.ashutoshgngwr.noice.MainActivity}" drawable="noice"/>
<!-- Photok -->
<item component="ComponentInfo{dev.leonlatsch.photok/dev.leonlatsch.photok.MainLauncher}" drawable="photok"/>
<!-- Randomix -->
<item component="ComponentInfo{com.minar.randomix/com.minar.randomix.activities.MainActivity}" drawable="randomix"/>
<!-- RBC Mobile -->
<item component="ComponentInfo{com.rbc.mobile.android/com.rbc.mobile.bud.pre_auth.common.activity.SplashActivity}" drawable="rbcmobile"/>
<!-- Sentien Launcher -->
<item component="ComponentInfo{com.sentienhq.launcher/com.sentienhq.launcher.MainActivity}" drawable="sentienlauncher"/>
<!-- Track & Graph -->
<item component="ComponentInfo{com.samco.trackandgraph/com.samco.trackandgraph.MainActivity}" drawable="trackgraph"/>
<!-- Transit Now -->
<item component="ComponentInfo{com.opl.transitnow/com.opl.transitnow.activity.stops.StopsActivity}" drawable="transitnow"/>
<!-- Uber Eats -->
<item component="ComponentInfo{com.ubercab.eats/com.ubercab.eats.app.feature.launch.LauncherActivity}" drawable="ubereats"/>
<!-- Vivint -->
<item component="ComponentInfo{com.vivint.vivintsky/com.vivint.vivintsky.ui.SplashActivity}" drawable="vivint"/>
<!-- Water Drink Reminder -->
<item component="ComponentInfo{com.northpark.drinkwater/com.northpark.drinkwater.SplashActivity}" drawable="waterdrinkreminder"/>
<!-- Weather -->
<item component="ComponentInfo{com.mbestavros.geometricweather/com.mbestavros.geometricweather.main.MainActivity}" drawable="weather"/>
<!-- Xeonjia -->
<item component="ComponentInfo{xyz.deepdaikon.xeonjia/xyz.deepdaikon.xeonjia.MainActivity}" drawable="xeonjia"/>
<!-- Zoysii -->
<item component="ComponentInfo{xyz.deepdaikon.zoysii/xyz.deepdaikon.zoysii.MainActivity}" drawable="zoysii"/>
```

**Comment by dalaeance** [comment]
Also, if you prefer limiting the amount of icons per request, I can break this up into smaller/individual requests. I'd imagine that'd be easier for tracking.

**Comment by Donnnno** [comment]
Thank you for the compliments!
It doesn't really matter how large the icon request is because it still can take a lot of time to add everything. 
I'll take a look at it when I have some time available (already completed tree other icon requests today).

**Comment by dalaeance** [comment]
Of course! 😊

Nice! I checked out the 3 you just completed and one of those includes apps I already have (but excluded from this request because I had seen those open issues). They look great; I'm looking forward to the next update!

**Comment by Donnnno** [comment]
Hi agian, 
I've completed the icon request. Here are some screenshots for the icons!
They should appear in the next update! (it's coming out very soon).
Enjoy :)

![image](https://user-images.githubusercontent.com/31142286/108242603-62dcf380-714d-11eb-9f17-c77809e6b9e6.png)
![image](https://user-images.githubusercontent.com/31142286/108242610-64a6b700-714d-11eb-9365-c15fe1494fe0.png)
![image](https://user-images.githubusercontent.com/31142286/108242619-66707a80-714d-11eb-99f3-6442df2af37d.png)

Ps. if you like them and want to support this icon pack, consider leaving a few bucks :) It can be very time-consuming to do this, thanks.

**Comment by dalaeance** [comment]
Thanks for doing this! This was way faster than I expected. I was thinking maybe another couple weeks.

Just a bit of feedback: If the "+" sign icon is for "Calculator++ (Window mode)", might I suggest taking the existing Calculator++ icon (already in your icon pack) and putting it in a box with a little "×" in the upper-right hand corner to represent a window's close button? There's already an app, "Track & Graph" that has the same plus sign above as its icon.

PS: I also sent you a donation for your work! I really appreciate what you're doing. ^_^

**Comment by Donnnno** [comment]
Hey! Thanks for the compliments, had some time extra. 
 I just saw the donation 😄 Thank you!!!!!

I'll take a look at the Calculator ++ (windowed mode icon) I'll probably change it in the next release :)

**Comment by dalaeance** [comment]
You're welcome! :relaxed: 

Okay, thanks. I'm looking forward to it. (:

**Comment by Donnnno** [comment]
![image](https://user-images.githubusercontent.com/31142286/108385211-bb72c600-720b-11eb-8a76-fe3330b58eb3.png)

something like this

**Comment by dalaeance** [comment]
TL;DR: Yes! This is what I had in mind! 😊
---

Yeah, something like this is what I was thinking. Though, I imagined it'd be the current icon for the main app/activity inside the window, so the "++" would be outlined pluses instead of strokes. 

That said, I think keeping them as strokes (as they are right now) may be a better way to go. I imagine the "++", as outlines, might look funny once they're shrunk down further and made to fit inside a window.

**Comment by dalaeance** [comment]
RE: The icon for Maps Go, it's looking like it's not being applied. Not sure if I should have made a separate issue for this, but figured I may as well comment in this thread.

```
<!-- Maps Go -->
<item component="ComponentInfo{com.google.android.apps.mapslite/org.chromium.webapk.shell_apk.MainActivity}" drawable="mapsgo"/>
```

**Comment by Donnnno** [comment]
Hmm, it should work tho, maybe in v1.4.0?

**Comment by dalaeance** [comment]
I tested it on both 1.3.5 and 1.4.0. Same result with it not being applied. I force-stopped both Arcticons and my launcher (KISS Launcher), and restarted my phone. No dice.

But, after doing a little investigating, it looks like the activity name portion has now changed from `org.chromium.webapk.shell_apk.MainActivity` to `com.google.maps.lite.twa.MapsLiteTwaLauncherActivity`.

```
<!-- Maps Go -->
<item component="ComponentInfo{com.google.android.apps.mapslite/com.google.maps.lite.twa.MapsLiteTwaLauncherActivity}" drawable="mapsgo"/>
```

**Comment by Donnnno** [comment]
Ah okey, I'll fix it in the next version ;)
